app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt

```diff
@@ -72,6 +77,7 @@ class MainActivity : SimpleActivity() {
     private var searchIndex = 0
     private var searchMatches = emptyList<Int>()
     private var isSearchActive = false
+    private val notesExporter by lazy { NotesExporter(this) }
```

Comment:
dont use lazy please, we arent using it anywhere

Reply:
Actually it's used in Simple SMS Messenger, so that's why I've used it: https://github.com/SimpleMobileTools/Simple-SMS-Messenger/blob/2f7cf12dc2ab45bae364fddadb920d1a0591f234/app/src/main/kotlin/com/simplemobiletools/smsmessenger/activities/MainActivity.kt#L50. However, I can change it.

Reply:
right, I didnt notice it has been added, wasnt me. Change it please, it is too unreadable to me.